tokio-util/src/net/mod.rs

```diff
@@ -51,7 +52,7 @@ pub struct ListenerAcceptFut<'a, L> {
     listener: &'a mut L,
 }
 
-impl<'a, L> Future for ListenerAcceptFut<'a, L>
+impl<L> Future for ListenerAcceptFut<'_, L>
```

Comment:
Which lint is this?

Reply:
```
warning: the following explicit lifetimes could be elided: 'a
  --> tokio-util/src/net/mod.rs:54:6
   |
54 | impl<'a, L> Future for ListenerAcceptFut<'a, L>
   |      ^^                                  ^^
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_lifetimes
   = note: `#[warn(clippy::needless_lifetimes)]` on by default
help: elide the lifetimes
   |
54 - impl<'a, L> Future for ListenerAcceptFut<'a, L>
54 + impl<L> Future for ListenerAcceptFut<'_, L>
```

Reply:
I think this lint does not improve the code. Please add `#![allow(clippy::needless_lifetimes)]` to the top of the `lib.rs` files instead to silence it.